### DISCUSSION TITLE
Is there a way to update groups by dragging and dropping a node inside one?

### DISCUSSION THREAD

**Discussion by dsaltares** [xyflow/xyflow#2060] [Q&A] [answered]
When I have a group, I would like to be able to add nodes to it by dragging and dropping them into the bounds of the group. I don't see any way to handle this in the API.

Is it possible? If so, how?

**Comment by yuniyakim** [comment, marked as answer]
I added the same functionality to my project.
I manually check if node is dropped inside any group by its' coordinates and if so, I set `parentNode` property of node to group id and calculate and set relative coordinates.
